fs/semaphore/Kconfig

```diff
@@ -11,9 +11,9 @@ config FS_NAMED_SEMAPHORES
 
 if FS_NAMED_SEMAPHORES
 
-config FS_NAMED_SEMPATH
+config FS_NAMED_SEMAPHORES_VFS_PATH
 	string "Path to semaphore storage"
-	default "/var/lock"
+	default "/var/sem"
```

Comment:
Is there any impact for applications on changing default value?

Reply:
Should not, because user call sem_unlink(not unlink) to remove the named semaphore from file system, and sem_unlink already update to use the new path.